DEVELOPMENT.md

```diff
@@ -188,29 +188,58 @@ Use `go test` to run the integration test suite:
 go test -v ./test/integration -tags=integration
 ```
 
+## Running ko builds with other base images
+By default `ko` uses a lightweight image to run go programs.
+The image can be [overridden](https://ko.build/configuration/#overriding-base-images) with
+the `defaultBaseImage` parameter of the `.ko.yaml` configuration file or through the
+environment variable  `KO_DEFAULTBASEIMAGE`.
+
+An example of an image with a set of debugging tools already installed that can be used is
+`registry.redhat.io/rhel9/support-tools`
+
+**[NOTE]**: Using this images usually needs the pod property `runAsNonRoot` set to `false`.
+
```

Comment:
I get the feeling that this block is telling me "if you need tools in your base container for something, run this command" but it does not provide me the command.

Is this section to shed some light in the fact that we are using the `KO_DEFAULTBASEIMAGE` for debugging down below, or is this aimed to help developers that need tooling in their base image?

If it aims to help developers that need tooling I would rename the section to "running with tools in the base image" and provide a clear command. If its to shed some light on `KO_DEFAULTBASEIMAGE` I don't think we need to explain this usage of `ko`, people can search for it.

Reply:
This section aims to help developers that need tooling, it isn't related to the usage of `KO_DEFAULTBASEIMAGE` when debugging.

I can add these commands:

```bash
export KO_DEFAULTBASEIMAGE=registry.redhat.io/rhel9/support-tools
```

and 
```
echo "defaultBaseImage: registry.redhat.io/rhel9/support-tools" > .ko.yaml 
```

to share the commands that could be used with the image that it's been recommended

Reply:
I think the `export` one is better as is temporary. The second one can lead to it being included into a PR and then merged by missing it.

Reply:
Added